graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/DeleteFetcher.java

```diff
@@ -101,7 +97,7 @@ protected Object get(
             Scope.DELETE,
             SourceAPI.GRAPHQL);
 
-    ResultSet resultSet = executeUnchecked(query, Optional.empty(), Optional.empty(), dataStore);
+    ResultSet resultSet = executeUnchecked(query, dataStore);
```

Comment:
All fetchers now override `getDatastoreParameters()` to set those arguments, instead of passing them here.